app/cdap/components/WranglerV2/RecipeStepsTable/index.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { IconButton, Typography } from '@material-ui/core';
+import { GridRowData } from '@material-ui/data-grid';
+import DeleteOutlinedIcon from '@material-ui/icons/DeleteOutlined';
+import {
+  GridContainer,
+  StyledDataGrid,
+  Wrapper,
+} from 'components/WranglerV2/RecipeStepsTable/styles';
+import React from 'react';
```

Comment:
React comes first in import order

Reply:
Just to be clear - external libraries should always be imported before internal components and with tsx files react should be imported first.

Reply:
updated import statements in the proper order